pkgs/development/python-modules/generic/default.nix

```diff
@@ -45,6 +45,8 @@
 # Additional flags to pass to "pip install".
 , installFlags ? []
 
+, format ? "setup"
```

Comment:
Thanks for your comments @adnelson 
See the first message about why I chose format like this.